.github/workflows/ci_docker.yml

```diff
@@ -0,0 +1,63 @@
+name: CI docker
```

Comment:
I think we should add another ci_ruby.yml file that runs the tests all the time and the docker workflow only when we want an image.

Reply:
There was already a ruby_ci.yml and ruby_build.yml; I can rename them.
The thing is, this Docker image doesnt change how we currently deploy the warehouse, they still need the archive.tar.gz to deploy. The Docker image for the moment is in use by the integration tests from other projects, so these 2 methods (archive.tar.gz and Docker) would be coexisting now.

Reply:
Should I rename ruby_ci? Or should I keep it as it is?

Reply:
Sorry I didn't realise that there were already workflow files for the repo. Since there are, they should be consistent, i.e. `docker_ci.yml`.

Reply:
Ok, renaming